CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 5.11.1(YYYY-MM-DD)
+
+### Enhancements
+* None.
+
+### Fixed
+* The Kotlin extensions library no longer defines a `app_name`, which in some cases conflicted with the `app_name` defined by applications. (Issue [#6536](https://github.com/realm/realm-java/issues/6536), since 4.3.0) 
```

Comment:
```suggestion
* The Kotlin extensions library no longer defines a `app_name`, which in some cases conflicting with the `app_name` defined by applications. (Issue [#6536](https://github.com/realm/realm-java/issues/6536), since 4.3.0) 
```